src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -113,6 +114,13 @@ public String getServicePhraseName(Interface service) {
     return Name.upperCamel(service.getSimpleName()).toPhrase();
   }
 
+  /** Artifact ID of the service */
+  public String getArtifactId(Interface service) {
+    List<String> tokens =
+        Splitter.on('.').splitToList(service.getModel().getServiceConfig().getName());
+    return tokens.size() > 0 ? "google-cloud-" + tokens.get(0) : null;
```

Comment:
Or it's probably better if it is provided at runtime from the library calling the GAPIC client.